models/Clients.ts

```diff
@@ -31,16 +19,16 @@ const clientsSchema = new Schema({
             ]
         }
     ],
-    finance: {
-        hourly: String,
-        totalEstimate: Number
-    },
+    originalCostEstimate: Number,
+    updatedCostEstimate: Number,
     project: {
         title: String,
         domain: String,
         externalDependencies: [String],
-        hoursTracked: Number,
-        notes: String
+        hoursLogged: Number,
+        notes: String,
+        originalLaunchDate: Date,
+        updatedLaunchDate: Date
```

Comment:
should we add the version into the project object ?

Reply:
No, no need to track it here. it gets tracked in the project package.json and Github